collectors/python.d.plugin/exim/README.md

```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # Exim monitoring with Netdata
 
-Simple module executing `exim -bpc` to grab exim queue.
+Simple module executing `exim -bpc` to grab the Experimental Internet Mailer (Exim) queue.
```

Comment:
I think we should just leave this as **Exim**. Seems like that's the real name now.